allensdk/test/brain_observatory/behavior/test_write_nwb_behavior_ophys.py

```diff
@@ -167,7 +167,7 @@ def test_add_partial_metadata(test_partial_metadata, roundtrip, roundtripper,
         identifier='afile',
         session_start_time=meta['experiment_datetime']
     )
-    nwb.add_metadata(nwbfile, meta)
+    nwb.add_metadata(nwbfile, meta, behavior_only=False)
```

Comment:
I think passing `behavior_only=False` can be omitted with a default value

Reply:
I don't think it hurts to be explicit?